src/utils/conversationCreator.js

```diff
@@ -5,7 +5,7 @@ const { CONVERSATION_MESSAGE_CONTEXT } = require('../models/conversations.model'
 const { getTransaction } = require('../blockchain/lib/web3Helpers');
 const {
   findSimilarDelegatedConversation,
-  findSimilarPayoutConversation,
+  findPayoutConversationByTxHashAndMilestoneId,
```

Comment:
Let's keep its name simple as before, arguments are self descriptive and don't need to be included in function name

Reply:
@aminlatifi  done.